packages/datx-jsonapi-angular/projects/datx-jsonapi-angular/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@datx/jsonapi-angular",
+  "version": "2.3.1",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
```

Comment:
Does this mean that if a project uses Angular 13, npm will complain? If so, then I would set the peer dependencies to >=12.